### DISCUSSION TITLE
Bing 與 ChatGPT 的認知戰

### DISCUSSION THREAD

**Discussion by ALAN30359704** [1101-GS4719-C/Python-Community-GS4719-C#1167] [Ideas]
Discussion options
不知道大家是否有過疑問，ChatGPT是否可以取代google 搜尋引擎的功能呢？
以下分享林宜敬老師貼出的一篇文章，
可以給我們一些解答與想法！
【Bing 與 ChatGPT 的認知戰】
一、
ChatGPT 能不能取代 Google 的搜尋引擎？不能，當然不能！因為兩者是不同類型的產品。
Google 搜尋引擎的專長是連上網際網路，幫我們找到我們需要的訊息。它沒有創造力，但只要我們使用得當，我們找到的資料大部分會是正確的。
而 ChatGPT 的專長是寫文章。它很有創造力，它的文筆很好。但它就像是一個不懂裝懂，想要在女生面前賣弄的高中男生屁孩一樣，遇到他不知道的事情，就會瞎掰出一個故事給我們。所以當我們問 ChatGPT問題的時候，得到的答案真真假假，錯誤很多。
Google 的搜尋引擎很厲害，ChatGPT 也很厲害。但兩者是不同類型的產品，所以 ChatGPT 不可能直接取代 Google 的搜尋引擎。

二、
但是微軟最近一直在宣傳，說他們即將把 ChatGPT 整合進他們的搜尋引擎 Bing 裡面，然後 Bing 就可以取代 Google 的搜尋引擎。
其實真正可以取代 Google 搜尋引擎的是 Bing，而不是 ChatGPT。微軟的目標是要用 ”Bing“ 取代 Google 的搜尋引擎，而不是要用 ”ChatGPT“ 取代 Google 的搜尋引擎。
微軟正在打一場認知戰。

三、
那微軟能不能將 ChatGPT 無縫地整合進 Bing 的搜尋引擎裡？
整合當然可以，但是要做到「無縫」很困難，因為 ChatGPT 跟搜尋引擎是不同類型的產品，兩者的功能不同。
當使用者想要用搜尋引擎找資料的時候，不會希望 ChatGPT 跑出來搶答；而當使用者想要問 ChatGPT 問題的時候，也不會希望搜尋引擎跑出來搶答。
而且，如果每次使用者用 Bing 搜尋引擎找資料的時候，ChatGPT都同時提供它的答案，那 ChatGPT 會忙死。然後由於 ChatGPT 所需要的運算能力遠超過搜尋引擎，所以微軟的營運成本會大幅提高。

四、
那微軟本身知不知道，ChatGPT 無法直接取代 Google 的搜尋引擎？
我想微軟當然知道。但是微軟也知道，一般人不知道。所以微軟就利用這個機會，讓一般人誤以為 ChatGPT 可以取代 Google 搜尋引擎，而新版的 Bing 就是一個結合了 ChatGPT 優點的，比 Google 更強大的搜尋引擎。
然後一般人就真的會使用微軟的 Bing 來取代 Google 的搜尋引擎。

五、
然後微軟還會讓大家知道，如果我們想要使用結合了 ChatGPT 功能的 Bing 搜尋引擎，那就必須先在 PC 上下載他們的 Edge 網路瀏覽器，或是在手機上下載他們的 Bing App。
因為微軟的目的，是要用 Microsoft Edge 網路瀏覽器來取代 Google Chrome 瀏覽器，或是用 Microsoft Bing App 來取代 Google Chrome App。

六、
那 Google 知不知道 ChatGPT 無法直接取代 Google 的搜尋引擎？
我想 Google 當然知道。但是 Google 也知道，一般人不知道，所以一般人會去下載使用微軟的 Edge 跟 Bing。
然後 Google 就會喪失他們在網路瀏覽器以及搜尋引擎的霸主地位。
所以 Google 很緊張，緊張到亂了手腳，急急忙忙推出了一個類似 ChatGPT，但是不成熟的產品。

七、
那華爾街的投資者知不知道 ChatGPT 無法直接取代 Google 的搜尋引擎？
我想他們應該也知道。但是華爾街的投資人同時知道，一般人不知道，所以一般人會去下載使用微軟的 Edge 跟 Bing。
然後微軟的股票就會大漲，Google 的股票就會大跌。

八、
看到這裡，你會覺得頭暈嗎？
如果你覺得頭暈的話，那就對了。因為認知作戰的初步目標，就是讓我們覺得頭暈。當我們頭暈之後，就會覺得微軟跟Google雙方講的都很有道理。
然後我們就會受到好奇心的驅使，趕快去下載 Edge 跟 Bing 來玩玩。
就像我一樣。

**Comment by Lipunpun** [comment]
頭真的暈爛，但我很認同這些觀點，Chatgpt的文章能力真的很強，但資訊正確程度真的很低，我認為Google是不會被他給取代，但如作者說的，被其他搜尋引擎取代我覺得是早晚的事（如果他沒有做出改變的話）

**Comment by DuxtinTai** [comment]
我也很期待 Bing 和 ChatGPT 的整合，但是這樣的整合並不意味著 Bing 就可以取代 Google 搜尋引擎。ChatGPT 在整合 Bing 搜尋引擎時可以提供更加人性化的答案，也可以更加方便地回答使用者的問題。但是，如果微軟想要真正挑戰 Google 的搜尋引擎，還需要進一步的努力和創新，不然一般人如我，並不會想要重新適應一種搜尋引擎。
